kruxstatsd/client.py

```diff
@@ -27,8 +44,16 @@ def __init__(self, prefix, host='localhost', port=8125, env=None):
 
     def _format(self, stat):
         """Format a stats string with the environment, prefix and hostname."""
-        return '%s.%s.%s.%s' % (
-            self.env, self.prefix, stat, self.hostname)
+        stat_list = [self.env, self.prefix]
+
+        if isinstance(stat, list):
+            stat_list += [str(s) for s in stat]
```

Comment:
I am using the list comprehension for `str(s)` here. I want this to be able to take list of integers and stuff.